### ISSUE TITLE
Add support for aws_mq_broker log settings

### ISSUE THREAD

**Issue by TimCostrop** [terraform-providers/terraform-provider-aws#5847]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The amazon MQ broker has two options to enable logging: general & audit, these options aren't available to be set via terraform atm.

### New or Affected Resource(s)
* aws_mq_broker

### Potential Terraform Configuration

```hcl
resource "aws_mq_broker" "my_mq_broker" {
  broker_name = "broker"
  engine_type = "ActiveMQ"
  engine_version = "5.15.0"
  host_instance_type = "mq.t2.micro"
  security_groups = []

  # optional
  logging {
    #default false for both
    general = true 
    audit = true
  }
}
```

**Comment by smita-kamat** [comment]
Merge this with https://github.com/terraform-providers/terraform-provider-aws/pull/6122

**Comment by bflad** [comment]
Support for a new `logs` configuration block has been merged in and will release with version 1.43.0 of the AWS provider, likely middle of next week. 👍

**Comment by bflad** [comment]
This has been released in [version 1.43.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1430-november-07-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
